### ISSUE TITLE
Disable "Materialize" for Type Greens when Contained

### ISSUE THREAD

**Issue by Rushi123456** [civilnetworks-projects/SCP_Public_Development#385]
https://www.civilgamers.com/community/threads/disable-materialize-for-type-greens-when-contained.13723/

Whenver Type-Greens are contained on their cells, they would not have access to "Materialize" ability.

**Comment by GarrySwepman** [comment]
Yeah one time when I was in HCZ I was shot by a T-G who was jamming himself inside the door to keep it open and it broke a arm, it really sucked.

**Comment by auburncivilnetworks** [comment]
Also disable their ability to reality bend players outside of their cells

**Comment by ZBuiX** [comment]
I imagine this can be done by making a permanent SRA radius within the TG Cells. Of course this is under the assumption that the TG CCs will remain the same.

Additionally, it is possible to implement a “ifbreached” variable to the Reality Bending swep to where it checks if you are breached or not. This 100% exists already. If Y, then you will be unable to use your reality bending powers. If N, then you can!

**Comment by Holland310** [comment]
marked to rewrite

**Comment by JamesDingleCN** [comment]
Closed as there is another suggestion which works better in the eyes of content which has been mentioned in the ticket previously (https://github.com/civilnetworks-projects/SCP_Public_Development/issues/611)
